src/components/templates/Camp/components/RoomList.tsx

```diff
@@ -33,7 +33,7 @@ export const RoomList: React.FunctionComponent<PropsType> = ({
       <div className="rooms-container">
         {rooms.map((room) => (
           <RoomCard
-            key={room.title}
+            key={`${venue.name}/${room?.title}`}
```

Comment:
Neither of these are unique. There technically isn't a unique value for rooms as far as we are aware.

Potential workaround, admin CLI tool that enumerates all rooms in a venue and generates a uuid (eg. with `uuid4()`), we could also add this to the logic on room creation, which would then ensure future rooms would have this as well.